files/scripts/sonic-netns-exec

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# Wrapper to execute any command in a specific
+# network namespace.
+# Usage:
+# sonic-netns-exec <namespace instance #> <command to be executed>
+
+NS="asic" #Network namespace
```

Comment:
>NS="asic" [](start = 0, length = 9)

This is a general script for any network namespace. So please don't assume namespace name start with `asic`.

The usage is like:
```
sonic-netns-exec <namespace name> <command to be executed> # will run on specified namespace
sonic-netns-exec '' <command to be executed> # will run on default namespace
```

Reply:
Fixed